posts/2020-12-18-rustdoc-new-contributors.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Rustdoc call for new contributors"
```

Comment:
The title is a bit "dry", I'll think of something more appealing, inspiring ... better.

Reply:
"You hate writing docs? So are we! Why not joining rustdoc then?" :rofl:

Reply:
> "You hate writing docs? So are we! Why not joining rustdoc then?" rofl

this title is pure gold, I hope I'll have a chance in the future to use it :D